### PR TITLE
Change Program.cs of Playground.Web to use WebApplication

### LINKED ISSUE #1579: Use `WepApplication` instead of HostBuilder for BlazorServer in the Playground project
In Bit Components Playground, web project,  we should use WebApplication in Program.cs rather than HostBuilder.

### PR DESCRIPTION
**Pull request by RezaKargar**
This closes #1579

### CODE REVIEW

**Comment by RezaKargar** [comment]
@ysmoradi Please check this out

**Comment by ysmoradi** [comment]
@RezaKargar 
The structure has 3 files
[Program.cs](https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Program.cs)
[Middlewares.cs](https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Startup/Middlewares.cs)
[Services.cs](https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Startup/Services.cs)
But you're submitting only 1 file.
The Todo template's web project is following Todo template's API project structure which has the same 3 files.

**Comment by RezaKargar** [comment]
@ysmoradi 
Oh, right!
I'll fix it

**Comment by ysmoradi** [comment]
This sounds good to me
Based on the discussion that we did, I ask you to handle this PR @msynk

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Program.cs [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
-namespace Bit.Client.Web.BlazorUI.Playground.Web
+namespace Bit.Client.Web.BlazorUI.Playground.Web;
+
+public class Program
 {
-    public class Program
-    {
-        public static async Task Main(string[] args)
-        {
 #if BlazorWebAssembly || BlazorServer
```
Comment (ysmoradi) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Program.cs:14]: Could you please use one Main method to be the same as TodoTemplate's web project?
Also add UseUrls

https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Program.cs#L12-L20

https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Program.cs#L40-L42

@RezaKargar

**Review by ysmoradi — CHANGES_REQUESTED** [review]
Please review and fix provided comments

**Review by ysmoradi — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**